fictadvisor-api/src/v2/api/controllers/TeacherController.ts

```diff
@@ -157,9 +185,22 @@ export class TeacherController {
     type: TeacherWithSubjectResponse,
   })
   @ApiBadRequestResponse({
-    description: `InvalidEntityIdException:\n
-                  teacher with such id is not found
-                  subject with such id is not found`,
+    description: `\n
+    InvalidEntityIdException:
+      Teacher with such id is not found subject with such id is not found`,
```

Comment:
All entities in InvalidEntityIdException must start with capital letter, so second text should look like: 
`Subject with such id is not found`